tremor-pipeline/src/lib.rs

```diff
@@ -73,6 +77,54 @@ pub use op::{ConfigImpl, InitializableOperator, Operator};
 pub use tremor_script::prelude::EventOriginUri;
 pub(crate) type ExecPortIndexMap =
     HashMap<(usize, Cow<'static, str>), Vec<(usize, Cow<'static, str>)>>;
+#[derive(Debug, Clone)]
+///Pluggable-logging Appender
+pub struct PluggableLoggingAppender {
+    /// async Sender
+    pub tx: Sender<LoggingMsg>,
+}
+
+impl Append for PluggableLoggingAppender {
+    fn append(&self, record: &log::Record) -> anyhow::Result<()> {
+        let path: String;
+        let lang = if let Some(optional_path) = record.module_path().map(ToString::to_string) {
+            path = optional_path;
+            if path == "tremor_pipeline::logging" {
+                LogSource::Tremor
+            } else {
+                LogSource::Rust
+            }
+        } else {
+            path = String::from("Path could not be retrieved");
+            LogSource::Rust
+        };
+
+        let vec = literal!({
+            "level":record.level().to_string(),
+            "args": record.args().to_string(),
+            "origin": lang.to_string(),
+            "path": path,
+            "file": record.file().map(ToString::to_string),
+            "line": record.line()
+        });
+
+        let msg = LoggingMsg {
+            language: lang,
+            payload: EventPayload::from(vec),
+            origin_uri: None,
+        };
+
+        println!("{:?}", msg);
+        block_on(self.tx.broadcast(msg))?;
+        Ok(())
+    }
+
+    fn flush(&self) {
+        todo!("TODO logging impl Append for PluggableLoggingAppender :: fn flush");
```

Comment:
Let's get rid of this `todo!` (I believe this function can be left empty, as the logs are going straight into the channel in `append`)

Reply:
this seems to be fixed, can we resolve the comment?